### PR TITLE
Change templates' specifications, remove view verb & add workflow ID

### PR DESCRIPTION
**Pull request by xtuchyna**

### CODE REVIEW

**Comment by goern** [comment]
why could this be removed?

/hold

**Comment by xtuchyna** [comment]
>why could this be removed?
It was added based on this PR #176 but we have found out that it was not the right solution.

**Comment by harshad16** [comment]
> why could this be removed?
> 
> /hold

i checked it, verb view is not present for template, also get would do the needful.

File: mi-scheduler/base/openshift-templates/mi-run-template.yaml [hunk 1]
```diff
@@ -3,19 +3,26 @@ kind: Template
...
+
 objects:
   - apiVersion: argoproj.io/v1alpha1
     kind: Workflow
     metadata:
-      generateName: mi-analysis-
+      name: mi-analysis-${WORKFLOW_ID}
       labels:
         app: scropsmetrics
     spec:
```
Comment (harshad16) [inline mi-scheduler/base/openshift-templates/mi-run-template.yaml:28]: Include arguments in spec as well.

```
arguments:
   parameters:
   - name: REPOSITORY
     value: ${REPOSITORY}
```

**Comment by harshad16** [comment]
Also can you add the below provided rolebinding in https://github.com/thoth-station/thoth-application/blob/142cc216331e4afb2dadcdcabba6704b7b10c34b/mi-scheduler/base/role_binding.yaml

```
---
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: RoleBinding
metadata:
  name: mi-scheduler-argo-admin
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: argo-admin
subjects:
  - kind: ServiceAccount
    name: mi-scheduler
```

File: mi-scheduler/base/role.yaml [hunk 2]
```diff
@@ -27,4 +27,4 @@ rules:
       - patch
       - update
       - watch
-      - view
+
```
Comment (harshad16) [inline mi-scheduler/base/role.yaml:30]: remove the extra newline

**Pull request merged**